episimlab/cy_utils/cy_utils.pyx

```diff
@@ -29,6 +29,14 @@ cdef gsl_rng *get_seeded_rng(int int_seed) nogil:
     gsl_rng_set(rng, int_seed)
     return rng
 
+# binomial distribution from GSL lib
+cdef extern from "gsl/gsl_randist.h" nogil:
+    unsigned int gsl_ran_binomial(gsl_rng * r, double p, unsigned int n)
+
+# logarithm (base e)
+cdef extern from "math.h":
```

Comment:
@eho-tacc I'm having some trouble with compilation, I believe due to typing issues with the complex numbers. Any suggestions?

Reply:
Reverting to poisson transitions resolves the typing issue by removing the need for complex numbers (and thus removing the need for me to figure out how to type them properly). I'll proceed with testing the new compartment structure with poisson transitions, and if tests pass and model results are sensible I think we can stick with poisson transitions.